api_app/script_analyzers/classes.py

```diff
@@ -213,6 +231,14 @@ def before_run(self):
         )
 
     def after_run(self):
+        # We delete the file only if we have single copy for analyzer
```

Comment:
It'd be better to have this logic be part of the `clean_old_jobs` celery task. We shouldn't immediately delete this file because of it might be needed in connectors.

Reply:
Mmmh no. If we do not delete the sample copies, we will instantly fill the disk space.
No race condition should be present, since each analyzer will have its own copy.

Moreover, connectors will use the main sample, the aws one, we do not actually need them anymore after the analysis is done